acquisitions/src/main/resources/thunderjet/mod-orders/features/bind-piece.feature

```diff
@@ -55,12 +55,11 @@ Feature: Verify Bind Piece feature
     * call createOrder { id: '#(orderId)' }
 
     # 3. Create patron and user
-    * def userGroup = 'GROUP0369';
     * def v = call createUserGroup { id: '#(patronId)' }
     * def v = call createUser { id: '#(userId)', patronId: '#(patronId)'}
 
     # 4. Setup Circulation Policy
-    * call createCirculationPolicy { id: '#(policyId)', policyName: '#(policyName)' }
```

Comment:
if we do not use policyId, policyName - we can remove these parameters from feature file

Reply:
thanks, removed.